ComposeSamples/README.md

```diff
@@ -0,0 +1,51 @@
+---
+page_type: sample
+name: Surface Duo - Jetpack Compose samples
+languages:
+- kotlin
+products:
+- surface-duo
+description: "Samples showing how to use Jetpack Compose to achieve dual-screen user interface patterns."
+urlFragment: all
+---
+# Surface Duo Jetpack Compose samples
+
+This repo contains the samples built in Jetpack Compose for Microsoft Surface Duo. These samples demonstrate [user interface patterns](https://docs.microsoft.com/dual-screen/introduction#dual-screen-app-patterns) with the new UI framework.
+
+![Jetpack Compose samples CI](https://github.com/microsoft/surface-duo-app-samples/workflows/App%20samples%20CI/badge.svg)
+
+## Getting Started
+
+To learn how to load apps on the Surface Duo emulator, see the [documentation](https://docs.microsoft.com/dual-screen/android), and follow [the blog](https://devblogs.microsoft.com/surface-duo).
+
+## Prerequisites
+
+- Jetpack Compose version: `1.0.0-beta04`
```

Comment:
Joy I see now that in `dependencies.gradle` we have a newer version of compose: `1.0.0-beta05`

Reply:
Yes, you are right. 👍